front-end/src/Speak.tsx

```diff
@@ -48,15 +50,9 @@ export default function Speak(props) {
 
     textHasBeenSpoken.current = true;
     if (ttsApiKey) {
-      playElevenLabsAudio();
-    } else {
-      const msg = new SpeechSynthesisUtterance();
```

Comment:
Remove old way of doing this - not worth it. If they don't have the API key, or if it fails, just don't do announcements.

Reply:
Added this back with a little refactoring